ovirt-ansible-collection.spec.in

```diff
@@ -11,11 +11,6 @@ License:	ASL 2.0 and GPLv3+
 BuildArch:	noarch
 Url:		http://www.ovirt.org
 
-%if 0%{?rhel} == 9
-Requires:	ansible-core >= 2.11.0
-%else
-Requires:	ansible >= 2.9.21
-%endif
```

Comment:
Please add:
```
# TODO: Require ansible-core when both engine and hosts are ready for it
# Requires:    ansible-core >= 2.12.0
```

Reply:
Changed the ansible version to 2.11. On el8 will probably not be ansible 2.12 because of python support.

Reply:
Let's try 2.12, I think Python 3.6 and 3.8 could coexist on the same host